packages/airnode-admin/src/cli.ts

```diff
@@ -514,12 +514,29 @@ yargs
       const airnodeAddress = await admin.deriveAirnodeAddress(mnemonic);
       const airnodeXpub = admin.deriveAirnodeXpub(mnemonic);
 
+      const mnemonicLength = mnemonic.length;
```

Comment:
Could we abstract this into a function and unit test the ASCII art output? It would be nice to ensure it's properly displayed.